tests/templates/kuttl/oidc/20-assert.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 600
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: oidc-login-test
+status:
+  ready: 1
```

Comment:
Hmm does `succeeded: 1` not work? Ready sounds to me like there is a Pod running for a few seconds, but not necessarily that it succeeded

Reply:
This is actually done to start log output of the test job only after the pod has been created. The "real" assert step is `21-assert` where we are waiting for `succeeded: 1`.

Reply:
done